extensions/liquibase/runtime/src/main/java/io/quarkus/liquibase/NativeImageResourceAccessor.java

```diff
@@ -0,0 +1,63 @@
+package io.quarkus.liquibase;
+
+import java.io.IOException;
+import java.net.URI;
+import java.nio.file.FileSystem;
+import java.nio.file.FileSystemAlreadyExistsException;
+import java.nio.file.FileSystems;
+import java.nio.file.Path;
+import java.util.Collections;
+import java.util.List;
+
+import org.jboss.logging.Logger;
+
+import liquibase.resource.AbstractPathResourceAccessor;
+import liquibase.resource.PathResource;
+import liquibase.resource.Resource;
+
+public class NativeImageResourceAccessor extends AbstractPathResourceAccessor {
```

Comment:
This one needs to be in the `runtime` subpackage as it's not API.

Reply:
done